wiki/addons/third-party/kubejs-tfc/page.kubedoc

```diff
@@ -0,0 +1,5 @@
+Download: [CurseForge](https://www.curseforge.com/minecraft/mc-mods/kubejs-tfc), [Modrinth](https://modrinth.com/mod/kubejs-tfc)
+
+---
+
+KubeJS TFC's wiki is hosted on its own website maintained by its owner, found [here](https://notenoughmail.github.io/kubejs_tfc/)
```

Comment:
I'm not sure if I like this. The page looks pretty blank right now - for example, you could give a short code snippet or an image so that others know what to expect.w

Reply:
This also applies to some other pages tho (thanks @Donaldino7712), so you _should_ be fine

Reply:
I had though about that, but couldn't come up with anything that didn't either feel out of place or promotional in some way, though if you have ideas I'd be happy to consider them